### ISSUE TITLE
HDMI-only: Black screen after power-on or bitstream loading

### ISSUE THREAD

**Issue by sy2002** [MJoergen/C64MEGA65#43]
Black screen from time to time on startup (independent if a cartridge is inserted or not): HDMI-only: VGA shows a normal C64 start screen including a blinking cursor. Problem:

* On HDMI the screen stays black
* But you can use the OSM via the Help key (??!)

Can be fixed by a short-reset.

**Comment by MJoergen** [comment]
@sy2002 : Do you have statistics on this bug, i.e. how often it happens ?

Has this been seen when loading image via JTAG ? Or is it only after power-on ?

**Comment by sy2002** [comment]
@MJoergen I have seen this via JTAG and via power-on but it happens infrequently / seldomly.

In contrast to that, issue https://github.com/MJoergen/C64MEGA65/issues/53 happens 100% of the time.

**Comment by sy2002** [comment]
Still happens, but happens so seldomly that we won't fix this for V5 and let's see if it's still worth an own issue in the future. Closing as wontfix.
